scalafmt-tests/src/test/resources/spaces/InterpolatedStringCurlyBracesAlways.stat

```diff
@@ -0,0 +1,21 @@
+spaces.inInterpolatedStringCurlyBraces = Always
+
+<<< Simple variable
+s"Hello ${the} world!"
+>>>
+s"Hello ${ the } world!"
+
+<<< Complex statement
+s"Hello ${th.e} world!"
+>>>
+s"Hello ${ th.e } world!"
+
+<<< Function call
+s"Hello ${the()} world!"
```

Comment:
can you add some tests with triple quotes and a newline after opening curly and/or before closing curly?

Reply:
Tried to add, but it seems to be broken in 2.7.5 already :smile: 
```
[info]   =======
[info]   => Diff
[info]   =======
[info]   -s"""Hello ${
[info]   -   |the
[info]   -   |} world!"""
[info]   +s"""Hello ${| the
[info]   +  |} world!""" (InterpolatedStringCurlyBraces.stat:13)
```